package.json

```diff
@@ -20,10 +20,11 @@
     "deploy-test": "npm run build && cd examples/test-studio && sanity deploy",
     "deploy-movies": "npm run build && cd examples/movies-studio && sanity deploy",
     "lint": "npm run eslint -- .",
-    "eslint": "node_modules/.bin/eslint --ext=.js,.jsx,.mjs,.ts,.tsx --quiet",
+    "eslint": "eslint --ext=.js,.jsx,.mjs,.ts,.tsx --quiet",
```

Comment:
Without something like npx won't this break if you haven't added `node_modules/.bin` to your path locally? 🙂

Reply:
It won’t break without `npx`, since `npm` will look for the executable in `node_modules/.bin` by default.

Reply:
https://stackoverflow.com/questions/33270880/run-npm-scripts-using-local-deps